week0003/truongngodang/java/Week0003.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Week0003 {
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+        String[] stringRequest = new String[12];
```

Comment:
How do you know there will be 12 requests? Why don't you assume 1 million requests?